app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/apps/NewAppBroadcastReceiver.kt

```diff
@@ -95,10 +93,6 @@ class NewAppBroadcastReceiver @Inject constructor(
         kotlin.runCatching { applicationContext.unregisterReceiver(this) }
     }
 
-    private fun isGame(packageName: String): Boolean {
-        return appCategoryDetector.getAppCategory(packageName) is AppCategory.Game && !appTpFeatureConfig.isEnabled(AppTpSetting.ProtectGames)
```

Comment:
`appCategoryDetector` and `appTpFeatureConfig` can be removed from the constructor.

Reply:
ah nice, will do